gcetcbendorsement/sevvalidate.go

```diff
@@ -31,6 +31,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const testonlyForceGCSGUID = "00000000-0000-0000-0000-000000000000"
```

Comment:
I can't figure out what this is supposed to do?

Reply:
The SNPValidate function has a peculiar design. For every entry of the certificate table that matches the given UUID, the CertTableOptions function gets called on the stored contents.

If there is _no_ entry with that UUID, and the entry is required, then function is still called this time with `nil` for the contents. This allows for a fallback to network. By using a UUID that we never assign in the certificate table, I'm ensuring that the network fallback happens.

Reply:
IMO it would be better to use a random UUID rather than an all-zeros one since that would have a lower chance of collision

Reply:
Done.